docs/source/building/windows.rst

```diff
@@ -18,19 +18,24 @@ Install dependencies:
         base-devel \
         cmake \
         diffutils \
+        doxygen \
         gcc \
         git \
         make \
         mingw-w64-x86_64-binutils \
         mingw-w64-x86_64-boost \
         mingw-w64-x86_64-cmake \
         mingw-w64-x86_64-curl \
+        mingw-w64-x86_64-graphviz \
         mingw-w64-x86_64-miniupnpc \
         mingw-w64-x86_64-nodejs \
         mingw-w64-x86_64-onevpl \
         mingw-w64-x86_64-openssl \
         mingw-w64-x86_64-opus \
-        mingw-w64-x86_64-toolchain
+        mingw-w64-x86_64-rust \
```

Comment:
rust is required to build a wheel for a python dependency needed to build the docs